### PR TITLE
Change oil in cake-recipe

### PR DESCRIPTION
**Pull request by offchan42**
Please change the oil ingredient.

### CODE REVIEW

File: cake-recipe.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canols oil
```
Comment (off99555) [inline cake-recipe.txt]: The pronunciation is wrong, change it before I will pull this.

Comment (off99555) [inline cake-recipe.txt]: dsfsdf

Comment (offchan42) [inline cake-recipe.txt]: Wrong pronunciation

File: cake-recipe.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup canols oil
+ - 1/2 cup canola oil
```
Comment (offchan42) [inline cake-recipe.txt:10]: That's right pronunciation. This is what I expected from you.

File: cake-recipe.txt [hunk 3]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canola oil
```
Comment (off99555) [inline cake-recipe.txt]: Nice!

**Pull request merged**